### PR TITLE
LOC CHECKIN | roslyn dev15.6.x | 20180202

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

**Comment by tmeschter** [comment]
VSO bug for Escrow: [561034](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/561034).

**Comment by tmeschter** [comment]
test windows_release_unit64_prtest please

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf [hunk 1]
```diff
@@ -7479,20 +7479,7 @@ Pokud chcete odstranit toto varování, můžete místo toho použít /reference
...
- /errorreport:&lt;řetězec&gt;        Určuje způsob zpracování interních chyb kompilátoru: 
-                               prompt, send, queue nebo none. Výchozí možnost 
-                               je queue (zařadit do fronty).
- /appconfig:&lt;soubor&gt;           Určuje konfigurační soubor aplikace, 
-                               který obsahuje nastavení vazby sestavení.
- /moduleassemblyname:&lt;řetězec&gt; Určuje název sestavení, jehož součástí bude 
-                               tento modul.
- /modulename:&lt;řetězec&gt;         Určuje název zdrojového modulu.
-</target>
+ /preferreduil</target>
```
Comment (tmeschter) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf]: This resource has been truncated somehow.

**Comment by v-zbsail** [comment]
PR updated with the fix

**Comment by natidea** [comment]
@tmeschter this was approved in https://devdiv.visualstudio.com/DevDiv/_workitems/edit/561034
Please merge when ready

**Comment by tmeschter** [comment]
@natidea Done.

**Pull request merged**